src/AmbientSounds.Uwp/Controls/TitleBarLogo.xaml.cs

```diff
@@ -12,14 +14,44 @@ public sealed partial class TitleBarLogo : UserControl
             typeof(TitleBarLogo),
             new PropertyMetadata(Strings.Resources.AppDisplayName));
 
+    public static readonly DependencyProperty DisplayTextForegroundBrushProperty =
+        DependencyProperty.Register(nameof(DisplayTextForegroundBrush),
+            typeof(Brush),
+            typeof(TitleBarLogo),
+            null);
+
     public TitleBarLogo()
     {
         this.InitializeComponent();
+        Window.Current.Activated += Window_Activated;
+        SetDisplayTextForeground(Window.Current.CoreWindow.ActivationMode != CoreWindowActivationMode.Deactivated);
     }
 
     public string DisplayText
     {
         get => (string)GetValue(DisplayTextProperty);
         set => SetValue(DisplayTextProperty, value);
     }
+
+    public Brush DisplayTextForegroundBrush
+    {
+        get => (Brush)GetValue(DisplayTextForegroundBrushProperty);
+        set => SetValue(DisplayTextForegroundBrushProperty, value);
+    }
+
+    private void SetDisplayTextForeground(bool isWindowFocused)
+    {
+        DisplayTextForegroundBrush = App.Current.Resources[isWindowFocused ?
+            "TextFillColorPrimaryBrush" : "TextFillColorTertiaryBrush"] as Brush;
```

Comment:
As mentioned in another comment, is it possible to re-implement this pr using visual states so we get the theme support automatically?

Reply:
Sure.